conftest.py

```diff
@@ -0,0 +1,15 @@
+# Configure test collection to skip doctests for modules that depend on sphinx
```

Comment:
I wonder if this is necessary. What about including `numpydoc[default]` in our test dependencies and then we can test everything?

Alternatively, what about marking the tests that require sphinx as expected fails when sphinx isn't available?

Reply:
> What about including numpydoc[default] in our test dependencies and then we can test everything?

This would work for CI but wouldn't work out-of-the-box for a development environment. I personally think there's value in being able to run *only* the tests (including doctests) for the non-sphinxy bits.

Reply:
It might be worth using nox if we want that because now we are talking about needing to use two different virtual environments to run the tests like this.